wazo_calld/plugins/calls/services.py

```diff
@@ -335,6 +335,20 @@ def unhold_user(self, call_id, user_uuid):
         self._verify_user(call_id, user_uuid)
         self.unhold(call_id)
 
+    def answer(self, call_id):
+        try:
+            channel = self._ari.channels.get(channelId=call_id)
+        except ARINotFound:
+            raise NoSuchCall(call_id)
+
+        protocol_interface = protocol_interface_from_channel(channel.json['name'])
+
+        self._phoned_client.answer_endpoint(protocol_interface.interface)
```

Comment:
phoned could could return `400` no plugin, `404` no such endpoint or `500`. We should do something here to avoid returning `500` 

I think there are some case of exception that wraps other service errors to return a 503 with the original error.